File: app/src/main/java/com/example/hw14app/MainActivity.java
package com.example.hw14app;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.view.View;
import android.view.View.OnClickListener;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClick1(View v) {
        Intent intent = new Intent(this, App1.class);
        startActivity(intent);
    }
    public void onClick2(View v) {
        Intent intent = new Intent(this, App2.class);
        startActivity(intent);
    }
    public void onClick3(View v) {
        Intent intent = new Intent(this, App3.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/hw14app/MainActivity.kt
package com.example.hw14app

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClick1(view: View) {
        val intent = Intent(this, App1::class.java)
        startActivity(intent)
    }

    fun onClick2(view: View) {
        val intent = Intent(this, App2::class.java)
        startActivity(intent)
    }

    fun onClick3(view: View) {
        val intent = Intent(this, App3::class.java)
        startActivity(intent)
    }

    fun onClick4(view: View) {
        val intent = Intent(this, App4::class.java)
        startActivity(intent)
    }
}